src/v3/src/components/PasswordWithConfirmation/PasswordWithConfirmation.tsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Box, FormHelperText } from '@mui/material';
```

Comment:
Can you also add more component layer unit test, like how field message is displayed with different event?

Reply:
I think a better test for this would be in the integration test because this component receives errors from the Form.tsx component upon submission which is out of scope for this test (for example, this component doesnt have a submit button so you cannot test submission).

Reply:
Sure thing will add unit test.